lib/chartmogul/metrics/activity.rb

```diff
@@ -29,12 +29,9 @@ class Activities < APIResource
 
       include Concerns::Entries
       include Concerns::PageableWithAnchor
+      include Concerns::PageableWithCursor
 
       set_entry_class Activity
-
-      def self.all(options = {})
```

Comment:
It was, we don't need it because the `Concern::Entries` already includes the `All` action here: https://github.com/chartmogul/chartmogul-ruby/blob/main/lib/chartmogul/concerns/entries.rb#L13